src/app/collection/pages/collection-502/collection-502.component.ts

```diff
@@ -7,9 +12,45 @@ import { Component, OnInit } from '@angular/core';
 })
 export class Collection502Component implements OnInit {
 
-  constructor() { }
+  learningObjects: LearningObject[];
+  guidelineNames: [];
+  loading = true;
+  query = {
+    limit: 5,
+    collection: '502_project'
+  };
+
+  constructor(
+    private navbarService: NavbarService,
+    private learningObjectService: LearningObjectService
+  ) { }
+
+  async ngOnInit() {
+
+    this.navbarService.show();
+    this.fetchLearningObjects(this.query);
 
-  ngOnInit(): void {
   }
 
+  async fetchLearningObjects(query: Query) {
+    this.loading = true;
+    this.learningObjects = [];
+    // Trim leading and trailing whitespace
+    query.text = query.text ? query.text.trim() : undefined;
+    try {
+      const {
+        learningObjects,
+        total
+      } = await this.learningObjectService.getLearningObjects(query);
+      this.learningObjects = learningObjects;
+      console.log(learningObjects);
+      this.loading = false;
+    } catch (e) {
+      console.log(`Error: ${e}`);
+    }
+  }
+
+  OnDestroy(): void {
+    this.navbarService.hide();
```

Comment:
Just curious, is this being used? Generally we use this for subscriptions but there's no subscription here

Reply:
I was copying the navbar implementation from another collection, but I don't think it's doing anything, so I've gone ahead and removed it.